containerd/Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN set -eux; \
 	rm -rf /var/lib/apt/lists/*
 
 # https://github.com/opencontainers/runc/releases
-ENV RUNC_VERSION 1.0.2
+ENV RUNC_VERSION 1.1.0-rc.1
```

Comment:
Might need to update the update.sh script here to ignore releases with `-rc` in them

Reply:
Great catch -- we didn't ignore them for a long time because all the releases were RCs, but I agree we should be ignoring them now. :sweat_smile: